README.rst

```diff
@@ -6,18 +6,17 @@ A job for scheduling solver to resolve dependency graphs and package-analyzer to
 Running the job locally
 =======================
 
-You can run this job locally. The job will talk to OpenShift API to schedule
-jobs in Thoth's middletier namespace and will check for packages which were not
-analyzed yet by talking to a cluster Dgraph's instance:
+You can run this job locally. Run a faust consumer and then run the faust producer with help of app.sh.
```

Comment:
Maybe link to thoth-messaging and the way to test a producer?

Reply:
Thanks, I have added that.